packages/backend/src/app/ee/authentication/enterprise-local-authn/enterprise-local-authn-module.ts

```diff
@@ -0,0 +1,6 @@
+import { FastifyPluginAsyncTypebox } from '@fastify/type-provider-typebox'
+import { enterpriseLocalAuthnController } from './enterprise-local-authn-controller'
+
+export const enterpriseLocalAuthnModule: FastifyPluginAsyncTypebox = async (app) => {
+    await app.register(enterpriseLocalAuthnController, { prefix: '/v1/authn/local' })
```

Comment:
I think we can omit `local` what do you think?

Reply:
local = username + password
federated = Google or Github (this will still come in the next PR isA)